cmd/weaviate/main.go

```diff
@@ -68,10 +48,28 @@ func main() {
 
 	switch opts.Target {
 	case "querier":
-		grpcQuerier, err := NewQuerier(log)
+		myConfig := config.GetConfigOptionGroup()
+		appState := rest.MakeAppState(context.TODO(), myConfig)
```

Comment:
creating `appState` this way is brilliant 👍

Reply:
i agree very good change. I kept wanting to avoid `configureAPI` cause it starts everything but we can actually re-use the make app state to avoid re-creating a ton of stuff!